routes/topics.js

```diff
@@ -4,15 +4,15 @@ const router = express.Router();
 
 // controller
 const topicsController = require('../controllers/topics');
-const {isAuth, isAuthV2} = require('../middlewares/auth');
+const {isAuth} = require('../middlewares/auth');
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from `isAuthV2` to `isAuth` in the import statement might have implications on how authentication is handled. If `isAuthV2` had additional or different functionality compared to `isAuth`, this could potentially introduce bugs or security issues. Please ensure that `isAuth` provides the necessary functionality and security checks that `isAuthV2` was providing.
```diff
- const {isAuth, isAuthV2} = require('../middlewares/auth');
+ const {isAuth} = require('../middlewares/auth');
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->